Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```diff
@@ -50,6 +50,11 @@ configuration:
   name: fetch_time
   required: false
   type: 0
+- defaultvalue: 7 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
```

Comment:
because commit fails with error : you dont have this value, please add to yml : 
- defaultvalue: 7 days
  display: First fetch time
  name: first_fetch
  required: false
  type: 0